double_with_map_handle_tc.rb

```diff
@@ -0,0 +1,14 @@
+array = [1,2,3,4,5,6,7,"jahsjdfhaj"]
+size = array.size
+
+array.map! do |i|
+
+    if(i.is_a? Numeric)
+        p "if block called #{i}"
+        i = i * 2
+    else
+        i = i
```

Comment:
@mohseengs why are you reassigning the `i` value?

Reply:
and remove the printing statements

Reply:
as i have used destructive version of map